pilot/proxy/envoy/testdata/cds-ssl-context.json.golden

```diff
@@ -165,6 +165,42 @@
      "verify_subject_alt_name": []
     }
    },
+   {
+    "name": "out.httpbin.default.svc.cluster.local|http",
+    "service_name": "httpbin.default.svc.cluster.local|http",
+    "connect_timeout_ms": 1000,
+    "type": "strict_dns",
+    "lb_type": "round_robin",
+    "hosts": [
+     {
+      "url": "tcp://httpbin.default.svc.cluster.local:80"
+     }
+    ],
+    "ssl_context": {
+     "cert_chain_file": "/etc/certs/cert-chain.pem",
+     "private_key_file": "/etc/certs/key.pem",
+     "ca_cert_file": "/etc/certs/root-cert.pem",
+     "verify_subject_alt_name": []
```

Comment:
here as well.. Basically you should ensure that calls to external services have no ssl context.. if its port 80, it should be in http connection manager. if its 443, then it should be in the tcp proxy such that you first filter based on service IP and then finally have an orig_dst_cluster that simply lets all traffic through to the destination via an orig_dst cluster that does not use auth